client/package.json

```diff
@@ -7,41 +7,43 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e"
+    "e2e": "ng e2e",
+    "e2etest": "npm run e2e -- --configuration=ci --webdriverUpdate=false"
```

Comment:
feel free to remove this command since we didn't get this to work.

Reply:
Ok, I removed it.